packages/desktop/components/popups/BalanceFinderPopup.svelte

```diff
@@ -34,19 +45,24 @@
     export let title: string
     export let body: string
 
-    const { isStrongholdLocked, network } = $activeProfile
-    const searchAlgorithm: SearchAlgorithmType =
-        network.id === NetworkId.Iota || NetworkId.IotaAlphanet ? SearchAlgorithmType.IDS : SearchAlgorithmType.BFS
-
     let error = ''
     let isBusy = false
+    let searchInCurrentWallet: boolean = false
+
+    const { isStrongholdLocked, network } = $activeProfile
 
     $: isTransferring = $visibleActiveAccounts.some(
         (account) => account.hasConsolidatingOutputsTransactionInProgress || account.isTransferring
     )
     $: searchForBalancesOnLoad && !$isStrongholdLocked && onFindBalancesClick()
     $: consolidateAccountsOnLoad && !$isStrongholdLocked && onConsolidateAccountsClick()
     $: totalBalance = sumBalanceForAccounts($visibleActiveAccounts)
+    $: searchInCurrentWallet, (hasUsedBalanceFinder = false)
```

Comment:
settings this to false here can lead to problems, in `onDestroy` we have an if with this, if the user searches and still touched this checkbox then there will no refreshes, we should fix this 🙏🏼